src/sympc/protocol/aby3/aby3.py

```diff
@@ -204,3 +206,78 @@ def _get_truncation_pair(x: MPCTensor, session: Session) -> Tuple[MPCTensor]:
         r_mpc = MPCTensor(shares=r, session=session, shape=x.shape)
         rPrime_mpc = MPCTensor(shares=rPrime, session=session, shape=x.shape)
         return r_mpc, rPrime_mpc
+
+    @staticmethod
+    def local_decomposition(
```

Comment:
Q: Does this work -- if we change from `ring_size == 2` to `ring_size = 2^32` does it holds?

Reply:
Yeah it works for conversion from conversion of ring_size from one to another.